src/EnrollTotpController.js

```diff
@@ -111,9 +110,15 @@ const EnrollTotpControllerEnrollTotpController = FormController.extend({
           View: EnrollTotpControllerAppDownloadInstructionsView,
           showWhen: showWhenDeviceTypeSelected,
         }),
-        FormType.Toolbar({
-          noCancelButton: true,
-          save: loc('oform.next', 'login'),
```

Comment:
Q: this triggers the submit operation on any input type other than 'submit' as well? and the current fix is to make the submit operation work only for when attribute is 'save'

Reply:
Yes